flexget/plugins/filter/crossmatch.py

```diff
@@ -83,8 +85,12 @@ def entry_intersects(self, e1, e2, fields=None, exact=True):
                 log.trace('field %s is not in both entries', field)
                 continue
 
-            v1 = e1[field]
-            v2 = e2[field]
+            if case_sensitive:
+                v1 = e1[field]
+                v2 = e2[field]
+            else:
+                v1 = e1[field].lower()
```

Comment:
I don't think a field value will always be a string in which case this will crash.

Reply:
I didn't realize it could be a value other than string.  What other types are valid?  Any suggestions are welcome

Reply:
int, date, bool, etc .. you need to add `if isinstance(e1[field], basestring):` in somewhere there and it's fine :)

Reply:
Except use `str`, not `basestring`. ;)

Reply:
Would it be possible for the value to be an array of strings?

Reply:
It is possible, I'm not sure if we really need to worry about that though.

Reply:
ok, made the changes.  let me know thx